lib/mercadopago.rb

```diff
@@ -300,6 +300,7 @@ def initialize(debug_logger=nil)
 				@http.use_ssl = true
 				@http.verify_mode = OpenSSL::SSL::VERIFY_PEER
 				@http.ssl_options = OpenSSL::SSL::OP_NO_SSLv3 # explicitly tell OpenSSL not to use SSL3

```

Comment:
Thanks @jvillarejo but we prefer `OpenSSL::SSL::SSL_OP_NO_TLSv1`

Reply:
No problem. I don't have problem to update it to ```OpenSSL::SSL::SSL_OP_NO_TLSv1``` Because I've received a message from mercadopago that I'm not using TLS > 1.0 and I was surprise as I was using this gem, but then I've checked the code and saw  this ```OpenSSL::SSL::OP_NO_SSLv3```

Reply:
**I used 2.3.3**

![captura de pantalla 2018-06-13 a la s 18 16 15](https://user-images.githubusercontent.com/864790/41378609-ec5c119e-6f35-11e8-8742-7f91c7747254.png)

For use more than one SSL option, can use an ssl_mask variable like:

```ruby 
options_mask = OpenSSL::SSL::OP_NO_SSLv3 + OpenSSL::SSL::OP_NO_TLSv1
@http.ssl_options = options_mask
```

**PD** Let me check the _Ruby_ and _openssl gem_ changelog for `OpenSSL::SSL::OP_NO_TLSv1` for ruby >= 2.5.x